sdk/translation/Azure.AI.Translation.Text/tests/Samples/SampleSnippets.cs

```diff
@@ -127,6 +127,24 @@ public TextTranslationClient CreateTextTranslationClientWithToken()
             return client;
         }
 
+        [Test]
+        public TextTranslationClient CreateTextTranslationClientWithAad()
+        {
+            #region Snippet:CreateTextTranslationClientWithAad
+
+#if SNIPPET
+            string endpoint = "<Text Translator Custom Endpoint>";
+#else
+            string endpoint = TestEnvironment.CustomEndpoint;
+#endif
+            DefaultAzureCredential credential = new DefaultAzureCredential();
+            TextTranslationClient client = new TextTranslationClient(credential, new Uri(endpoint));
```

Comment:
This will use the existing `TokenCredential` constructor just fine.  There's no need for a specific overload.